vuepress/docs/next/tutorials/consume/solr.md

```diff
@@ -3,81 +3,77 @@ sidebarDepth: 2
-* [Helm](https://helm.sh/docs/intro/install/) to handle Solr installation
+* [Helm](https://helm.sh/docs/intro/install/) to handle the Solr installation
 
 ## Install and Configure Solr 
  
-1. Install Solr Helm charts  
-```
+1. Install Solr Helm charts:  
+``` bash
 helm repo add apache-solr https://solr.apache.org/charts
 helm repo update
 ```  
-For more information, see the [Solr Getting Started page](https://solr.apache.org/guide/7_3/solr-tutorial.html).
-
-2. Install the Solr and Zookeeper CRDs. **Note:** creating the CRDs and installing the operators is done at the cluster level and will require cluster-level permissions.
-```
+2. Install the Solr and Zookeeper CRDs:  
+**Note:** creating the CRDs and installing the operators are done at the cluster level and will require cluster-level permissions.
+``` bash
 kubectl create -f https://solr.apache.org/operator/downloads/crds/v0.5.0/all-with-dependencies.yaml
 ```
-3. Install the Solr and Zookeeper operators
-```
+3. Install the Solr and Zookeeper operators:
+``` bash
 helm install solr-operator apache-solr/solr-operator --version 0.5.0
 ```
-4. Download the template `entando-solrCloud.yaml`. You can adjust the resource settings using this file, e.g., memory, cpu, storage, replicas.
+4. Download the template `entando-solrCloud.yaml`. Adjust the resource settings such as memory, CPU, storage, or replicas using this file.
 
 <EntandoCode>curl -sLO "https://raw.githubusercontent.com/entando/entando-releases/{{ $site.themeConfig.entando.fixpack.v72 }}/dist/ge-1-1-6/samples/entando-solrCloud.yaml"</EntandoCode>
 
-5. Create the Solr application resources
-```
+5. Create the Solr application resources:
+``` bash
 kubectl apply -f entando-solrCloud.yaml -n YOUR-NAMESPACE
-```
-4. Check that the service has started properly:
-```
+``` 
+6. Check that the service started properly:
+``` bash
 kubectl get solrclouds -w
 ```
 You should see a response similar to this:
-```
+``` bash
 NAME   VERSION   TARGETVERSION   DESIREDNODES   NODES   READYNODES   UPTODATENODES   AGE
 solr   8                         3              3       3            3               79m
 ```
 
 ## Generate the Core
 
 1. Set up port forwarding to the Solr common service:
-```
+``` bash
 kubectl port-forward service/solr-solrcloud-common 8983:80
 ```
 **Note:** `kubectl port-forward` should be left running.
 
-2. Open another terminal and call the Solr API to generate the Solr core collection. **Note:** the first core (e.g., for the primary tenant in a multitenant environment, or the only core in a single tenant environment) must use "entando" instead of "YOUR-TENANT-ID" for the collection name.
-
-   | Placeholder | Description |
-   |:--|:-- |
-   | YOUR-TENANT-ID | The identifier for the tenant, e.g., mysite1 |
-
+2. Open another terminal and call the Solr API to generate the Solr core collection.    
+**Note:** 
+    * For the core name, use `YOUR-TENANT-ID` = `entando` except for secondary tenants  
```

Comment:
1. I think these two lines are less clear in this version since both lines speak to multitenancy.  
2. I think there could be some benefit in using the same table format for each of these tutorials so it's very easy to see the definition of the placeholders across related tutorials. In that case calling out the exception in the table might be an alternative.

Reply:
@jyunmitch I see @avdev4j sent some changes in this area. It might need one more tweak for readability but your call there.